apps/web/src/services/alert-manager-v2/store/service-create-form-store.ts

```diff
@@ -47,6 +47,15 @@ export const useServiceCreateFormStore = defineStore('service-create-form', () =
         initState() {
             state.currentStep = 1;
             state.currentSubStep = 1;
+            state.step1Form = {
+                name: '',
+                key: '',
+                member: {},
+                description: '',
+            };
+            state.selectedWebhookType = {} as PluginModel;
```

Comment:
Rather than using `as`, how about writing the type of `selectedWebhookType` as `Partial<PluginModel>`?

Reply:
Partical<PluginModel> 
or
PluginModel | {}